app/assets/javascripts/short-form/components/applicationSummary.js.coffee

```diff
@@ -40,7 +40,7 @@ angular.module('dahlia.components')
         $translate.instant('label.file_attached', {file: $translate.instant('label.verification_letter')})
 
       ctrl.prioritiesSelectedExists = ->
-        _.isEmpty(ctrl.application.adaPrioritiesSelected)
+        !_.isEmpty(ctrl.application.adaPrioritiesSelected)
```

Comment:
we were showing "household details" section in the summary on applications that didn't have ADA priorities, and not showing it on applications that had those questions

Reply:
:+1: